package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@hmcts/postcodeinfo-client",
+  "version": "0.0.1",
+  "description": "MoJ postcode lookup client",
+  "license": "MIT",
+  "repository": "https://github.com/hmcts/postcodeinfo-client-node",
+  "files": [
+    "lib"
+  ],
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib && rimraf coverage",
+    "format": "prettier --write \"{src,__tests__}/**/*.ts\" --single-quote --trailing-comma es5",
+    "lint": "tslint --project tsconfig.json --format verbose",
+    "prebuildOnly": "yarn clean && yarn format && yarn lint && echo Using TypeScript && tsc --version",
+    "build": "tsc --pretty",
+    "test": "jest",
+    "coverage": "jest --coverage",
+    "watch": "yarn build --watch",
+    "watch:test": "jest --watch"
+  },
+  "dependencies": {
+    "request": "^2.83.0",
+    "request-promise-native": "^1.0.5"
+  },
+  "devDependencies": {
+    "@types/jest": "^21.1.8",
+    "@types/nock": "^8.2.1",
+    "@types/node": "^8.0.0",
+    "@types/request-promise-native": "^1.0.10",
+    "@types/supertest": "^2.0.4",
+    "jest": "^21.2.1",
+    "nock": "^9.1.4",
+    "prettier": "^1.5.2",
+    "rimraf": "^2.0.0",
+    "sinon": "^4.1.2",
+    "supertest": "^3.0.0",
+    "ts-jest": "^21.2.4",
+    "ts-node": "^3.2.0",
```

Comment:
should this not be 4.0.0 now that it is released?

Reply:
hmm could be 4.0, but there's a quite major bug on it that caused us to downgrade the citizen-frontend, so will hold off